sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -220,20 +191,13 @@ export interface Receiver<ContextT> {
     entityPath: string;
     entityType: "queue" | "subscription";
     getDeadLetterPath(): string;
-    getMessageIterator(options?: GetMessageIteratorOptions): AsyncIterableIterator<{
-        message: ReceivedMessage;
-        context: ContextT;
-    }>;
+    getMessageIterator(options?: GetMessageIteratorOptions): AsyncIterableIterator<ReceivedMessageT>;
     isReceivingMessages(): boolean;
-    receiveBatch(maxMessages: number, maxWaitTimeInSeconds?: number, options?: ReceiveBatchOptions): Promise<{
-        messages: ReceivedMessage[];
-        context: ContextT;
-    }>;
-    receiveDeferredMessage(sequenceNumber: Long, options?: OperationOptions): Promise<ServiceBusMessage | undefined>;
-    receiveDeferredMessages(sequenceNumbers: Long[], options?: OperationOptions): Promise<ServiceBusMessage[]>;
+    receiveBatch(maxMessages: number, maxWaitTimeInSeconds?: number, options?: ReceiveBatchOptions): Promise<ReceivedMessageT[]>;
```

Comment:
`maxWaitTimeInSeconds` should go in `ReceiveBatchOptions`

Reply:
Agreed, will move.